### PR TITLE
homepage fix (i fucked up the route)

### PR DESCRIPTION
**Pull request by ta5one**

### CODE REVIEW

**Review by caleb-love — CHANGES_REQUESTED** [review]
Fix these better!

File: index.html [hunk 1]
```diff
@@ -22,9 +22,9 @@ <h1>Welcome to "We Love CSS"!</h1>
         <h2>Our Members</h2>
         <p>Click on a member's name to view their page:</p>
         <ul>
-            <li><a href="/mo-ta-ali.html">Mohammed T Ali</a></li>
```
Comment (caleb-love) [inline index.html:25]: What case type are we using?

**Comment by ta5one** [comment]
> Fix these better!

they're fixed perfectly

**Review by ta5one — COMMENTED** [review]
this is great

**Pull request merged**